### PR TITLE
feat: 🍰 [WIP][SubPR] Implement Org Creation (Single Person Org)

### LINKED ISSUE #4802: 🚀 [Feature] Implement Creation Of Orgs
## :rocket: Feature Request

Implement creation of orgs.

### PR DESCRIPTION
**Pull request by Tirokk**
## 🍰 Pullrequest

Implement creation of orgs.

### Issues

- fixes #4802

### Todo

- [ ] use the list of social media as template
  - [x] refactor social media list #4771
  - [ ] fix Cypress test for social media deletion in `UserProfile.SocialMedia`
- [ ] use user settings for creation and listing of the users orgs
  - [ ] put the setting of an org there underneath if you edit it

#### Clean Up

- [ ] search for `Wolle`

#### Org Settings Data

- [ ] name
- [ ] slug (partly autogenerated)
- [ ] org category in short (theme-channel, artist name, self-employee, group, company)
  - free edit field (with example text or suggestion menu?)
- [ ] about
- [ ] location
- [ ] social media

#### Org Funktiontionallities In Settings

- [ ] link to org profile if org already exists

#### Org Funktiontionallities On Profile (not common with User profile)

- [ ] show it’s an org an not a user
- [ ] show the orgs category
- [ ] display the admin(s) of the org (?)

#### May Create Own Issues Or Move To Others

- [ ] user profile shows their own orgs and memberships

### CODE REVIEW

File: backend/src/schema/types/type/Org.gql [hunk 1]
```diff
@@ -0,0 +1,96 @@
+type Org implements Profile {
```
Comment (Mogge) [inline backend/src/schema/types/type/Org.gql:1]: Did we agree in not using subcalsses for groups?

File: backend/src/schema/types/type/Org.gql [hunk 2]
```diff
@@ -0,0 +1,96 @@
...
+  # Profile interface properties begin
+  # a real type inheritance would be possible via: https://github.com/nicolasdao/graphql-s2s
+  
+  id: ID!
+  slug: String!
+  name: String
+  actorId: String
```
Comment (Mogge) [inline backend/src/schema/types/type/Org.gql:8]: What is this?

File: webapp/components/_new/features/MySomethingList/MySomethingList.spec.js [hunk 3]
```diff
@@ -114,13 +133,45 @@ describe('MySomethingList.vue', () => {
...
           const deleteButton = wrapper.find('.base-button[data-test="delete-button"]')
           deleteButton.trigger('click')
           await Vue.nextTick()
-          const expectedItem = expect.objectContaining({ id: 'id', dummy: 'dummy' })
-          expect(propsData.callbacks.delete).toHaveBeenCalledTimes(1)
-          expect(propsData.callbacks.delete).toHaveBeenCalledWith(expect.any(Object), expectedItem)
+          // const expectedItem = expect.objectContaining({ id: 'id', dummy: 'dummy' })
+          // expect(propsData.callbacks.delete).toHaveBeenCalledTimes(1)
+          // expect(propsData.callbacks.delete).toHaveBeenCalledWith(expect.any(Object), expectedItem)
+          const expectedModalData = expect.objectContaining({
```
Comment (Mogge) [inline webapp/components/_new/features/MySomethingList/MySomethingList.spec.js:143]: I would prefer this in the expect statement

File: webapp/components/_new/features/MySomethingList/MySomethingList.spec.js [hunk 4]
```diff
@@ -114,13 +133,45 @@ describe('MySomethingList.vue', () => {
...
+                    callback: expect.any(Function),
+                  },
+                },
+              },
+            },
+          })
+          expect(mutations['modal/SET_OPEN']).toHaveBeenCalledTimes(1)
+          expect(mutations['modal/SET_OPEN']).toHaveBeenCalledWith(
+            expect.any(Object),
+            expectedModalData,
```
Comment (Mogge) [inline webapp/components/_new/features/MySomethingList/MySomethingList.spec.js:173]: Place the whole object here. If the test fails, you have to look up for the variable

**Review by Mogge — CHANGES_REQUESTED** [review]
I am not sure about this. I suggest the following
 * Avoid the complex mixed classes of `profile -> org` and `profile -> user`. Let us keep this as simple as possible. We never worked with this and we cannot know the problems we will face.
 * Let us use a new `Group` model for the groups
 * Let us avoid refactoring in other components, Go straight forward. Later, when we have more time, we can refactor for efficiency and synergies
 * Route `/group/id/slug` for the profile of a group
 * Route `/groups` for the list of groups
 * Copy the user profile to group, to have a base to work on
 * Use the Post template to create/edit Group as blueprint
 * KISS

**Comment by Tirokk** [comment]
This is now realized by groups

**Pull request closed without merging**